src/data/reducers/assets.js

```diff
@@ -59,21 +111,36 @@ const status = (state = {}, action) => {
   }
 };
 
-const parameters = (state = initialState, action) => {
+export const request = (state = requestInitial, action) => {
   switch (action.type) {
-    case assetActions.FILTER_UPDATED:
-      return { ...state, assetTypes: { ...state.assetTypes, ...action.data } };
     case assetActions.SORT_UPDATE:
-      return { ...state, ...action.data };
+      return {
+        sort: action.data.sort,
+        direction: action.data.direction,
+      };
+    case assetActions.PAGE_UPDATE:
+      return {
+        ...state,
+        page: action.data.page,
+      };
+    case assetActions.FILTER_UPDATED:
+      return {
+        ...state,
+        assetTypes: { ...state.assetTypes, ...action.data },
+      };
     default:
       return state;
   }
 };
 
-const assets = combineReducers({
+export const assets = combineReducers({
```

Comment:
This is sort of a relic left over from refactoring. I don't know whether it's really necessary to wrap the list in a combineReducers function? Should I leave it or look into just exporting list? Or exporting only list but also renaming it to assets?

Reply:
It seems like exporting assets as just list should work (`export const assets = list;`)

Reply:
Do you think that that's necessary? I feel like at that point I should just rename the list reducer to be assets. What do you think?

Reply:
Oh true, that does sound better.